flow360/component/simulation/validation/validation_simulation_params.py

```diff
@@ -39,6 +46,103 @@ def _check_consistency_wall_function_and_surface_output(v):
     return v
 
 
+def _check_cht_solver_settings(v):
+    has_heat_transfer = False
+
+    models = v.models
+
+    if models:
+        for model in models:
+            if isinstance(model, Solid) and model.entities is not None:
+                has_heat_transfer = True
+            if isinstance(model, Wall) and model.heat_spec is not None:
+                has_heat_transfer = True
+
+        if has_heat_transfer is False:
+            v = _validate_cht_no_heat_transfer(v)
+        if has_heat_transfer is True:
+            v = _validate_cht_has_heat_transfer(v)
+
+    return v
+
+
+def _validate_cht_no_heat_transfer(v):
+
+    # for model in v.models:
+    #     if isinstance(model, Solid) and model.heat_equation_solver:
+    #         raise ValueError("Heat equation solver activated with no zone definition.")
+
+    # for model in v.models:
+    #     if isinstance(model, Wall) and isinstance(model.heat_spec, Temperature):
+    #         raise ValueError(
+    #             "SolidIsothermalWall boundary is defined with no definition of volume zone of heat transfer."
+    #         )
```

Comment:
These are no longer needed. We allow iso thermal wall but not "SolidIsothermalWall"